locale/en/download/package-manager.md

```diff
@@ -59,20 +59,20 @@ pacman -S nodejs npm
 Node.js is available as a module called `nodejs` in CentOS/RHEL 8 and Fedora.
```

Comment:
@billbalm, thanks for the PR &mdash; this looks like a good catch, but cannot actually confirm this for myself at the moment&hellip;

@richardlau may be the most appropriate person able to review this PR and ensure that this is correct. From what I gathered from the previous Release WG meeting, there is expected loss of support for CentOS/RHEL 7 for Node 18.